package.json

```diff
@@ -16,13 +16,15 @@
     "babel-loader": "^8.0.5",
     "jest": "^26.6.3",
     "kremling": "^2.0.4",
+    "kremling-loader": "2.1.1-beta.0",
```

Comment:
If kremling-loader's changes are merged in, we shouldn't need this beta tag anymore, right?

Reply:
correct, i fixed this 👍